packages/rolldown/src/binding.d.cts

```diff
@@ -843,8 +850,12 @@ export interface Helpers {
   mode?: HelperMode
 }
 
-/** TypeScript Isolated Declarations for Standalone DTS Emit */
-export declare function isolatedDeclaration(filename: string, sourceText: string, options?: IsolatedDeclarationsOptions | undefined | null): IsolatedDeclarationsResult
+/**
+ * TypeScript Isolated Declarations for Standalone DTS Emit (async)
+ *
+ * Note: This function can be slower than `isolatedDeclarationSync` due to the overhead of spawning a thread.
+ */
+export declare function isolatedDeclaration(filename: string, sourceText: string, options?: IsolatedDeclarationsOptions | undefined | null): Promise<IsolatedDeclarationsResult>
```

Comment:
Not a strong opinion, but if the naming is not intentional, it's better to name it `isolatedDeclarations` following the community convention.

https://github.com/rolldown/rolldown/pull/6961#discussion_r2534287684

<img width="1668" height="2042" alt="Image" src="https://github.com/user-attachments/assets/35103186-106c-4dd3-a6f1-c15d54f70922" />